docs/usage.md

```diff
@@ -35,15 +35,43 @@ three main sections, one each for configurations, images, and session templates.
 These sections may be specified in any order, and any of the sections may be
 omitted if desired.
 
-#### Creating CFS configurations
+#### Bootprep input file schema version
```

Comment:
Do you think it could even be shortened to "Providing a schema version"? It's a subsection of "Editing a bootprep input file" already, so I think the context is fairly clear.

Reply:
Yes, I think that's a great idea!